management/client.go

```diff
@@ -110,6 +110,9 @@ type Client struct {
 
 	// If `true` then the client will require Pushed Authorization Requests
 	RequirePushedAuthorizationRequests *bool `json:"require_pushed_authorization_requests,omitempty"`
+
+	// URLs that are valid to call back from Auth0 for OIDC backchannel logout.
+	OIDCBackchannelLogout *OIDCBackchannelLogout `json:"oidc_backchannel_logout,omitempty"`
```

Comment:
What happens if a user tries to set this value but the tenant doesn't have the feature flag? Is it obvious what to do from the error message? If not could we append some additional info here to alert the user that the feature flag needs to be enabled on the tenant until this is GAed?

Reply:
Setting this property will return `Failed 400 Bad Request: Payload validation error: 'Additional properties not allowed: oidc_backchannel_logout`, but attempting to set `RequirePushedAuthorizationRequests` will return `403 Forbidden: The account is not allowed to perform this operation, please contact our support team.`

I'll look to update both of these to include a note around ensuring they are enabled